<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit</class>
 <widget class="QDialog" name="edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="2">
    <widget class="QPushButton" name="increaseMovingRatio">
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_18">
     <property name="text">
      <string>M Ratio</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="label_19">
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="HorizontalRotationToLeft">
     <property name="text">
      <string>LEFT</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_27">
     <property name="text">
      <string>Z Ratio</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_24">
     <property name="text">
      <string>H ROTATE</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QPushButton" name="decreaseHRotationAngle">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_20">
     <property name="text">
      <string>0.2</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="decreaseVRotationAngle">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>V angle</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>H angle</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="increaseZoomRatio">
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="decreaseMovingRatio">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QPushButton" name="decreaseZoomRatio">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="HorizontalRotationToRight">
     <property name="text">
      <string>RIGHT</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="increaseVRotationAngle">
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>0.1</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="label_28">
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QPushButton" name="increaseHRotationAngle">
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="VerticalRotationToLeft">
     <property name="text">
      <string>LEFT</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="VerticalRotationToRight">
     <property name="text">
      <string>RIGHT</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>V ROTATE</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
